### PR TITLE
new example code - printing all btcusd quotes

### PR DESCRIPTION
**Pull request by artursapek**
quotes

new demo file showing how to consume trades from all markets of a given pair (BTCUSD in this example)

### CODE REVIEW

File: examples/btcusd-quotes/btcusd-quotes.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+	c, err := streamclient.NewStreamConn(&streamclient.StreamParams{
+		URL: "wss://stream.cryptowat.ch",
+
+		Reconnect:        true,
+		ReconnectTimeout: 1 * time.Second,
+		Backoff:          true,
+		Subscriptions: []string{
+			"pairs:9:trades",
```
Comment (dimonomid) [inline examples/btcusd-quotes/btcusd-quotes.go]: I'd make this `9` and `148` below as separately defined constants, and comment them about how to get those ids, just to help readers of that example.

Comment (artursapek) [reply examples/btcusd-quotes/btcusd-quotes.go]: good idea

File: examples/print-quotes/print-quotes.go [hunk 2]
```diff
@@ -35,6 +37,17 @@ func main() {
 		marketSymbols = make(map[uint64]string)
 	)
 
+	if *pair == "" {
+		panic("Need -pair arg")
```
Comment (dimonomid) [inline examples/print-quotes/print-quotes.go]: I don't think it's worth to panic on a wrong user input, with stack traces etc; just an error message would be better.

Comment (dimonomid) [reply examples/print-quotes/print-quotes.go]: So that one got ignored.
Ugh, ok, not a biggie. :)

**Pull request merged**